pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "encoded"
-version = "1.3.3"
+version = "1.3.3abcd"
```

Comment:
This is a bug fix that hopefully won't change the perceived action of the system.
I would call it 1.3.4.